core/build.gradle

```diff
@@ -127,10 +127,12 @@ dependencies {
             'org.postgresql:postgresql:42.5.2',
             'com.vladmihalcea:hibernate-types-52:2.21.1',
             'com.mchange:c3p0:0.9.5.5',
+            "org.springframework:spring-aspects:5.3.25",
```

Comment:
Is this needed or did you forget it ?

Reply:
Yes, i't written in the doc that's a requirement, and without it, it's not working...